src/content/docs/data-apis/custom-data/aws-privatelink.mdx

```diff
@@ -200,6 +214,12 @@ These are the New Relic endpoint services available via AWS PrivateLink:
         </tbody>
       </table>
 
+      <Callout variant="important">
+        Review the following constraints when configuring the `identity-api.newrelic.com` and `infrastructure-command-api.newrelic.com` hostnames:
+        - These are only exposed in the `us-east-2` (Ohio) region.
+        - The endpoint service does not have an associated DNS private name associated. Create a PrivateLink connected to this service endpoint, and create the Private Hosted Zone (PHZ) for each hostname. 
```

Comment:
```suggestion
        - The endpoint service does not have an associated DNS private name. Create a PrivateLink connected to this service endpoint, and create the Private Hosted Zone (PHZ) for each hostname. 
```

Reply:
Another way to say this would be like this: "The endpoint service does not have a DNS private name associated with it."

I don't understand Private DNS enough to be confident here.